test/e2e/common/host_path.go

```diff
@@ -86,12 +90,20 @@ var _ = framework.KubeDescribe("HostPath", func() {
 		subPath := "sub-path"
 		fileName := "test-file"
 		retryDuration := 180
+		hostPath := "/tmp"
 
 		filePathInWriter := path.Join(volumePath, fileName)
 		filePathInReader := path.Join(volumePath, subPath, fileName)
 
+		if selinux.SelinuxEnabled() {
+			filePath := path.Join(hostPath, subPath)
+			if err := relabelPathByChcon(filePath); err != nil {
+				glog.Warningf("%v", err)
```

Comment:
Would `t.Logf` be better here? You also might want to include some context ("Error relabeling %q: %v", filePath, err)